innovation.game.php

```diff
@@ -14287,7 +14322,7 @@ function stInterInteractionStep() {
                         $color = self::getGameStateValue('color_last_selected');
                         $selectable_colors = self::getAuxiliaryValueAsArray();
                         $selectable_colors = array_diff($selectable_colors, array($color)); // Remove the color of the card the player has chosen: he could not choose the same for his next card
-                        self::setAuxiliaryValueFromArray($selectable_colors);
```

Comment:
I think this is an artifact from being out of sync of dev? I did not make this change, I will have to remove it.

Reply:
I'm confused how this change from other commit ended up in here if you followed our dev flow. Yeah, please remove this.

Reply:
This is because I pulled/merged main into my branch but did not push my branch to my working project. So the changes are picked up when I push my working project to my branch.

Reply:
except it still it still has it.... 

I pulled -> push branch to innovationlordmcfuz -> made changes -> push innovationlordmcfuzz to branch -> review changes -> commit

Reply:
My guess is that you accidentally did `git pull` before you moved your code back over. But it's okay. I fixed it up.